docs/xblock-tutorial/reusable/create_db.rst

```diff
@@ -10,13 +10,5 @@ database.
 
    .. code-block:: none
 
-      (venv) $ python xblock-sdk/manage.py migrate
+      (xblock-env) $ python xblock-sdk/manage.py migrate
 
-#. You are prompted to indicate whether or not to create a Django superuser.
```

Comment:
I deleted this because I was not prompted to create a Django superuser.

<img width="1102" alt="image" src="https://github.com/openedx/XBlock/assets/1985317/045ecefd-b832-4ee6-9eb8-cbeb2e06c804">